zend_abstract_interface/zai_string/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_library(zai_string INTERFACE)
+
+target_include_directories(zai_string INTERFACE
+                                      $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+                                      $<INSTALL_INTERFACE:include>)
+
+target_compile_features(zai_string INTERFACE c_std_99)
+
+target_link_libraries(zai_string INTERFACE "${PHP_LIB}")
+
+add_library(Zai::Zai_string ALIAS zai_string)
```

Comment:
Nit: Is `Zai::Zai_string` to match the directory name? If so, could we make this `Zai::String` to match the rest of the components? (e.g. `Zai::Assert` is in the `zai_assert` directory.)